lib/mixpanel-node.js

```diff
@@ -143,16 +148,16 @@ var create_client = function(token, config) {
             }
             try {
                 body = JSON.parse(body);
+                if (body.status === 0) {
```

Comment:
`JSON.parse(null) = null`, it doesn't throw an error. I was thinking more of a `if (!body || body.status === 0)`